DataFormats/OnlineMetaData/interface/OnlineBeamSpotRecord.h

```diff
@@ -0,0 +1,73 @@
+#ifndef DATAFORMATS_ONLINEMETADATA_ONLINEBEAMSPOTRECORD_H
+#define DATAFORMATS_ONLINEMETADATA_ONLINEBEAMSPOTRECORD_H
+
+//---------------------------------------------------------------------------
+//!  \class OnlineBeamSpotRecord
+//!  \brief Class to contain online beamspot from soft FED 1022
+//!
+//!  \author Remi Mommsen - Fermilab
+//---------------------------------------------------------------------------
+
+
+#include <cstdint>
+#include <ostream>
+
+#include "DataFormats/OnlineMetaData/interface/OnlineMetaDataRaw.h"
+#include "DataFormats/Provenance/interface/Timestamp.h"
+
+
+class OnlineBeamSpotRecord
```

Comment:
I unerstand why we need `onlineMetaData::BeamSpot_v1` to store in the FED, but I'm wondering why we need an additional data format for the unpacked beamspot ?
Can we reuse `DataFormats/BeamSpot/interface/BeamSpot.h` instead ?

Reply:
@mommsen thanks.
I'm not an expert on he beamspot, I'd rather let @sarafiorendi or @rmanzoni comment about the CovarianceMatrix.

As far as I know the HLT does not use the time stamp of the beamspot.

Reply:
Hi, 
apologies I haven't really followed all this discussion.
Can you brief me up as to what is this new code intended for, so that we can answer sensibly?

Reply:
@rmanzoni, @mommsen is writing the unpacker for a software FED that should replace SCAL.

Rather than introducing a new data format, I suggested that the unpacker could produce directly a reco::BeamSpot object.

However we don't know exactly how the data from SCAL matches the members of  reco::BeamSpot .

Reply:
AFAIK, this code has been used for converting the SCAL beamspot data to reco::BeamSpot:
http://cmslxr.fnal.gov/source/RecoVertex/BeamSpotProducer/plugins/BeamSpotOnlineProducer.cc#0061